src/build-actions.js

```diff
@@ -82,7 +82,10 @@ const getWebpackConfig = async (actionPath, root, tempBuildDir, outBuildFilename
 
   // we have 1 required plugin to make sure is present
   config.plugins = config.plugins || []
-  config.plugins.push(new webpack.DefinePlugin({ WEBPACK_ACTION_BUILD: 'true' }))
+  config.plugins.push(new webpack.DefinePlugin({
+    WEBPACK_ACTION_BUILD: 'true',
+    'process.env.AIO_ENV_IS_STAGE': process.env.AIO_ENV_IS_STAGE || false
```

Comment:
wow very nice to do this with `webpack` !! 🙂 

I think we should set `process.env.AIO_CLI_ENV` which is parsable by `aio-lib-env` that could be used then in the libs to retrieve the env.

Maybe something like this could do:
```
// import aio-lib-env in `build-actions`
'process.env.AIO_CLI_ENV': getCliEnv() || DEFAULT_ENV
```
This will set `AIO_CLI_ENV` to stage if `AIO_CLI_ENV=stage aio app deploy` or if `.aio.cli.env === stage`.

Then in a runtime action `state.init()` and `files.init()`would call `aio-lib-core-tvm` that then could use `aio-lib-env.getCliEnv()` again to parse `process.env.AIO_CLI_ENV` set by webpack and call stage tvm if stage